doc/modules/linear_model.rst

```diff
@@ -1058,33 +1063,94 @@ the "saga" solver is usually faster.
 For large dataset, you may also consider using :class:`SGDClassifier`
 with `loss="log_loss"`, which might be even faster but requires more tuning.
 
-.. topic:: Examples:
+.. _liblinear_differences:
```

Comment:
```suggestion
.. _liblinear_differences:

Differences between solvers
^^^^^^^^^^^^^^^^^^^^^^^^^^^

There might be a difference in the scores obtained between
:class:`LogisticRegression` with ``solver=liblinear`` or
:class:`~sklearn.svm.LinearSVC` and the external liblinear library directly,
when ``fit_intercept=False`` and the fit ``coef_`` (or) the data to be predicted
are zeroes. This is because for the sample(s) with ``decision_function`` zero,
:class:`LogisticRegression` and :class:`~sklearn.svm.LinearSVC` predict the
negative class, while liblinear predicts the positive class. Note that a model
with ``fit_intercept=False`` and having many samples with ``decision_function``
zero, is likely to be a underfit, bad model and you are advised to set
``fit_intercept=True`` and increase the ``intercept_scaling``.
```